src/index.js

```diff
@@ -62,7 +65,8 @@ const now = function() {
 
 const logToFile = function(input) {
   const timestamp = Math.floor(Date.now() / 1000); //eslint-disable-line
-  const fileName = args.logdir + '/' + timestamp + '_' + uuid.v1() + '.json';
+  const fileName =
+    config.cliArgs.logdir + '/' + timestamp + '_' + uuid.v1() + '.json';
```

Comment:
Variable `args` is already an alias for `config.cliArgs` used throughout the whole file, we should probably keep it for consistency.